src/Airport.js

```diff
@@ -0,0 +1,49 @@
+const Plane = require('./Plane');
+
+class Airport {
+
+    planesAtAirport = [];
+
+    constructor(airportCapacity = 10) {
+        this.airportCapacity = airportCapacity;
+    }
+
+    landPlane(plane, weather) {
+        if (weather === `sunny` && plane instanceof Plane) {
+            if (!this.isFull() && !this.isPlaneAtAirport(plane.getID())) {
+                this.planesAtAirport.push(plane)
+            }
+        }
+    }
+
+    planeTakeOff(plane, weather) {
+        if (this.isPlaneAtAirport(plane.getID()) && weather === `sunny`) {
+            const index = this.planesAtAirport.indexOf(plane);
+            if (index > -1) {
+                this.planesAtAirport.splice(index, 1);
+            }
+        }
+    }
+
+    isPlaneAtAirport(planeID) {
+        for (let i = 0; i < this.planesAtAirport.length; i++) {
+            if (this.planesAtAirport[i].getID() === planeID) {
+                return true;
+            }
+        } return false;
+    }
+
+    increaseCapacity = capacity => {
+        this.airportCapacity += capacity;
+    }
+
+    decreaseCapacity = capacity => {
+        this.airportCapacity -= capacity;
+    }
+
```

Comment:
**decreaseCapacity method** - After submitting for grading I realised that I did not factor in a condition that, if the capacity goes below the planesAtAirport length, it should throw an error. On review I would have either added this condition into the decreaseCapacity method, or alternatively made one method to replace increaseCapacity and decreaseCapacity that would reset the current capacity, along with having a condition that will throw an error if the new capacity was below planesAtAirport length.